helios/pipeViewer/argos_dumper/ArgosCollection_test.cpp

```diff
@@ -117,17 +117,17 @@ int main()
 
     sparta::collection::PipelineCollector pc("testPipe", 0, root_node.getClock(), &root_node);
 
-    sparta::Scheduler::getScheduler()->finalize();
+    sched.finalize();
 
     pc.startCollection(&root_node);
 
     pc.printMap();
 
-    sparta::Scheduler::getScheduler()->run(100000);
+    sched.run(100000);
     pc.stopCollection();
-    sparta::Scheduler::getScheduler()->run(100000);
+    sched.run(100000);
     pc.startCollection(&root_node);
-    sparta::Scheduler::getScheduler()->run(100000);
+    sched.run(100000);
```

Comment:
think it would be possible to add pipeViewer collection to the example pipeline? it'd make for a very useful demo to be able to run a pipeline, collect data and view it all, step-by-step, in pipeViewer without having to write any custom code...

Reply:
Definitely!  It's on my LOOOOOOOOONG list of stuff to do.  I'm looking at basic documentation first, then a tutorial, followed by a FAQ, followed by using Helios (pipeViewer and Plato).